### PR TITLE
:rocket: :technologist: Document block redesign  #640

### PR DESCRIPTION
**Pull request by kiplangat01**
# Description

this PR fixes issue #640 Document block redesign

# Screenshot 

Screenshot from 2023-10-14 00-25-21
Screenshot from 2023-10-14 00-25-28

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/conv-learning-manager/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1033,7 +1033,11 @@
       "NAME":"Name",
       "EMAIL":"Email",
       "LAST-UPDATED":"Last Active",
+<<<<<<< HEAD
+      "ROLE":"Role",
+=======
       "ROLE":"Role(s)",
+>>>>>>> 8cb27d53dec924c58b28fa27ef681ebdfb99c03f
```
Comment (LemmyMwaura) [inline apps/conv-learning-manager/src/assets/i18n/en.json:1040]: Your PR has merge lines. This shouldn't be present

File: libs/features/convs-mgr/conversations/chats/src/lib/components/chat-card/chat-card.component.ts [hunk 2]
```diff
@@ -57,6 +65,60 @@ export class ChatCardComponent implements OnChanges, AfterViewInit, OnDestroy
...
+    this._sbs.sink = this._msgsQuery$.getLatestMessage(this.chat.id).pipe(
+      tap(latestMessage => {
+  
+          switch (latestMessage.type) {
+            case MessageTypes.TEXT: 
+              this.lastMessage = latestMessage.text;
+              break;
+            case MessageTypes.QUESTION:
+              this.lastMessage = latestMessage.options[0].optionText;
+              break;
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/conversations/chats/src/lib/components/chat-card/chat-card.component.ts:79]: Chats-related changes shouldn't be present. Kindly sync as this change was removed.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync your changes

**Comment by kiplangat01** [comment]
i have made a new pr #812 to solve this issue

**Comment by kiplangat01** [comment]
i have made a new pr https://github.com/italanta/elewa/pull/812 to solve this issue

**Comment by LemmyMwaura** [comment]
Closing this as it's been resolved.

**Pull request closed without merging**